controllers/fluentdconfig_controller.go

```diff
@@ -114,35 +113,35 @@ func (r *FluentdConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	for _, fd := range fluentdList.Items {
-		// Get the selector contained in this fluentd instance
+		// Get the config selector in this fluentd instance
 		fdSelector, err := metav1.LabelSelectorAsSelector(&fd.Spec.FluentdCfgSelector)
 		if err != nil {
-			// Patch this fluentd instance if the selectors exsits errors
+			// Patch this fluentd instance if the selectors exsit errors
 			if err := r.PatchObjectErrors(ctx, &fd, err.Error()); err != nil {
 				return ctrl.Result{}, err
 			}
 		}
 
-		// A secret loader supports method to store the targeted fluentd config to the fd namespace, the the fd instance can share it.
+		// A secret loader supports LoadSecret method to parse the targeted secret.
 		sl := plugins.NewSecretLoader(r.Client, fd.Namespace, r.Log)
 
-		// pgr acts as a global plugins to store the related plugin resources
+		// pgr acts as a global resource to store the related plugin resources
 		pgr := fluentdv1alpha1.NewGlobalPluginResources("main")
 
 		// Firstly, we will combine the defined global inputs.
-		// Each cluster/namespace fluentd config will generate its own filters/outputs plugins with its cfgId/cfgLabel,
-		// and finally they would combine here together.
+		// Each cluster/namespace scope fluentd config will generate its own filters/outputs plugins with its cfgId/cfgLabel,
+		// and finally they would be combined.
 		pgr.CombineGlobalInputsPlugins(sl, fd.Spec.GlobalInputs)
 
 		// globalCfgLabels stores cfgLabels, the same cfg label is not allowed.
 		globalCfgLabels := make(map[string]bool)
 
-		// combine cluster cfgs
+		// Combine the resources which the cluster cfgs selected into pgr
```

Comment:
=>
		// Combine the resources matching the FluentdClusterConfigs selector into gpr